screens/Dashboard.js

```diff
@@ -6,14 +6,19 @@ import colorMap from '../utils/color';
 import {firebase} from '../utils/firebase';
 import 'firebase/database';
 import moment from 'moment';
+import * as Google from 'expo-google-app-auth';
+import SocialButton from "../components/CommonCompGoogleSignIn"
+import {firebaseConfig} from '../config';
+import googleSignout from '../screens/Login';
 
 class Dashboard extends Component {
     constructor(props) {
       super(props);
 
       this.state = {
         groups: [],
-        loading: false
+        loading: false,
+        userId: firebase.auth().currentUser ? firebase.auth().currentUser.uid : "testAdminId"
```

Comment:
we actually don't need the testAdminId from now on since we are not testing in the web client anymore.